lib/hydra/works/models/generic_work.rb

```diff
@@ -0,0 +1,6 @@
+module Hydra::Works
+  class GenericWork < ActiveFedora::Base
+    include Hydra::PCDM::ObjectBehavior
```

Comment:
**remove**. this is (and should be) included by Hydra::Works::GenericWorkBehavior